### DISCUSSION TITLE
Audio waveform graphs

### DISCUSSION THREAD

**Discussion by jorgegvallejo** [Abhinandan-Kushwaha/react-native-gifted-charts#284] [Ideas]
Hey, great work on this project. Just wanted to throw out there that it would be pretty cool if you all supported audio waveform graphs. Maybe a chart component where we just feed it an audio file?

**Comment by Abhinandan-Kushwaha** [comment]
@jorgegvallejo I think you should use a different package to convert audio file into a chart data. Then pass the resultant chart data in the chart library to render the Audio waveform graph.
